src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const allStates = [];
+  let copyState = Object.assign({}, state);
```

Comment:
Avoid reassigning variables, it increases code [readability](https://blog.sapegin.me/all/avoid-reassigning-variables/)